src/Spryker/Zed/Install/Business/InstallFacade.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spryker\Zed\Install\Business;
+
+use Spryker\Zed\Install\Communication\CommandLine\CommandLineArgumentContainer;
```

Comment:
Same. Mb we can move those classes to shared layer if we also need them in the communication one?